dev/functional/tests/cypress/integration/02-institution/04-manage-location.spec.ts

```diff
@@ -150,31 +110,19 @@ describe("Manage Locations", () => {
         .phoneInputText()
         .type(data.phoneNumber)
         .clear();
+
       institutionManageLocationObject
-        .institutionCodeErrorMessage()
-        .should("be.visible");
-      institutionManageLocationObject
-        .locationNameErrorMessage()
-        .should("be.visible");
-      institutionManageLocationObject
-        .addressErrorMessage()
-        .should("be.visible");
-      institutionManageLocationObject.cityErrorMessage().should("be.visible");
-      institutionManageLocationObject.postalErrorMessage().should("be.visible");
-      institutionManageLocationObject.stateErrorMessage().should("be.visible");
-      institutionManageLocationObject
-        .countryErrorMessage()
-        .should("be.visible");
-      institutionManageLocationObject
-        .firstNameErrorMessage()
-        .should("be.visible");
+        .errorMessage()
+        .eq(0)
+        .should("include.text", "First name is required");
       institutionManageLocationObject
-        .lastNameErrorMessage()
-        .should("be.visible");
-      institutionManageLocationObject.emailErrorMessage().should("be.visible");
+        .errorMessage()
+        .eq(1)
```

Comment:
As we talked about, this (`.eq(1)`) is subject to change and it will lead to a test error. It would be better to find a more flexible way. If it will not be taken care of in this PR please create a plan to talk about it.